hooks/pre-commit

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
```

Comment:
I think all we need are the imports 
```
from __future__ import print_function
from future import standard_library

standard_library.install_aliases()
from distutils.version import LooseVersion
import os
import re
import sys
import subprocess
import urllib.request
```
I have tested this on both python 3 and 2 and everything works fine for our work flow.(A PR where a script is changed should not have more than one version changes in various commits. We also emphasize that a PR should have one commit unless otherwise.)  Other edge cases can be handled by bypassing the pre-commit hook.

Thanks @zhangcandrew for this work